src/browser/modules/Stream/CypherFrame/helpers.js

```diff
@@ -353,13 +358,16 @@ const arrayifyPath = (types = neo4j.types, path) => {
 
 /**
  * Converts a raw Neo4j record into a JSON friendly format, mimicking APOC output
+ * Note: This preservers Neo4j integers as objects because they can't be guaranteed
```

Comment:
`preservers` should probably be preserves
`indentify` should probably be identify